test/topology_custom/test_old_ip_notification_repro.py

```diff
@@ -21,7 +21,7 @@ async def test_old_ip_notification_repro(manager: ManagerClient) -> None:
     """
     Regression test for #14257.
     It starts two nodes. It introduces a sleep in raft_group_registry::on_alive
```

Comment:
this commit needs to be adjusted (`raft_group_registry::on_alive`)

Reply:
it was supposed to indicate the error injection point
but if we're changing it as per the above I'll change it here too

Reply:
done in v5